app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_result",
+    [
+        (14, 14, [0, 0]),  # 14 cat years should convert into 0 human age, 14 dog years should convert into 0 human age.
```

Comment:
Add test cases to test against negative numbers, zero, large numbers and numbers that differ for cat and dog age according to the checklist